index.html

```diff
@@ -280,20 +280,22 @@ <h4 class="connect">LET'S CONNECT</h4>
                     <h1>Contact me</h1>
                     <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
                 </div>
-                <form id="form" method="POST" action="https://formspree.io/f/xjvzplnw">
+
+               <form id="form" method="POST" action="https://formspree.io/f/xjvzplnw">
                     <ul class="form-ul">
                         <li>
-                            <input type="text" name="name" id="user-name" placeholder="Enter your name here" required>
+                            <input type="text" maxlength="30" name="name" id="user-name" placeholder="Enter your name here" required>
                         </li>
                         <li>
                             <input type="email" name="email" id="email-address" placeholder="Enter your email here" required>
                         </li>
                         <li>
-                            <textarea name="message" id="msg" maxlength="500" placeholder="leave your message here " required></textarea>
+                            <textarea name="message" id="msg" maxlength="500" placeholder="Write your message here " required></textarea>
                         </li>
                         <span id="form-error" class="contact-top-p"></span>
-                        <li>
-                            <button type="submit" id="form-button">Get In Touch</button>
+                        <li class="btn">
```

Comment:
Great Job Team, Only one suggestion here. According to the Figma design the `Get In Touch` button is aligned to the left, straight to other input fields. In your case, you have centered it. Please try to mimic the figma design for a better layout. 

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/229887352-8cf8cd4d-dfe2-4e10-9076-08cc2220e506.png)

### Your design 
![image](https://user-images.githubusercontent.com/69077061/229887603-79acc79c-3a4d-4180-bf1c-2cd9835ccd64.png)